apiserver/src/test/java/org/dependencytrack/persistence/jdbi/ProjectDaoTest.java

```diff
@@ -229,6 +223,11 @@ public void testCascadeDeleteProject() {
         assertThat(notificationRule.getProjects()).isEmpty();
         qm.getPersistenceManager().refresh(policy);
         assertThat(policy.getProjects()).isEmpty();
+
+        // Ensure that metrics have been deleted.
+        MetricsDao dao = jdbiHandle.attach(MetricsDao.class);
+        assertThat(dao.getProjectMetricsSince(project.getId(), Instant.now())).isEmpty();
+        assertThat(dao.getDependencyMetricsSince(component.getId(), Instant.now())).isEmpty();
```

Comment:
The `Instant`s used here should be in the past, otherwise the method calls are guaranteed to never return anything.